### ISSUE TITLE
Double Datepicker raised when requestFocus has been called

### ISSUE THREAD

**Issue by daviddfa** [jifalops/datetime_picker_formfield#31]
For the version 0.1.8 when requestFocus is called from a another FormField and with editable: false configuration there are two DatePicker raised

```                     
TextFormField(
                      decoration: InputDecoration(
                        labelText: "Field 1",
                      ),
                      focusNode: _Field1FocusNode,
                      controller: _Field1Controller,
                      onEditingComplete: () => FocusScope.of(context)
                          .requestFocus(_date1FocusNode),
                    ),
                    DateTimePickerFormField(
                      inputType: InputType.date,
                      focusNode: _date1FocusNode,
                      controller: _date1Controller,
                      format: DateFormat('yyyy-MM-dd'),
                      editable: false,
                      initialDate: DateTime.utc(DateTime.now().year - 40,
                          DateTime.now().month, DateTime.now().day),
                      decoration:
                          InputDecoration(labelText: "Fecha de nacimiento"),
                    ),
```

**Comment by sante85** [comment]
+1
with flutter 1.7.8-hotfix.3 same error is raised.

Please resolve or workaround

**Comment by sante85** [comment]
return DateTimePickerFormField(
      firstDate: DateTime.now(),
      inputType: InputType.date,
      format: DateFormat('dd-MM-yyyy'),
      editable: false,
      validator: (DateTime value) {
        if (value == null) {
          return 'Data obbligatoria';
        } else {
          widget.orderDate = value;
        }
      },
      decoration:
          InputDecoration(labelText: 'Data *', hasFloatingPlaceholder: false),
    );

**Comment by jifalops** [comment]
This should be fixed in v0.3.0.
